bin/get_current_version.sh

```diff
@@ -1,19 +1,21 @@
 #!/bin/bash
-#This script will extract the current version based on the github repository's tags
+# This script will extract the current version based on the github repository's tags
 
-#Get current_version
+# Get current_version
 export CURRENT_VERSION
 CURRENT_VERSION=v$(git describe --tags --always | tr -d .)
 
-regex='^v[0-9]{3}([ab][0-9]{1,}|rc[0-9]{1,})?$'
+export KUBENOW_REGEX
```

Comment:
Actually no. I though I could have re-used it, but was not the case then. Removing the extras `export` lines.